Python.gitignore

```diff
@@ -133,3 +133,11 @@ dmypy.json
 
 # pytype static type analyzer
 .pytype/
+
+# IDE
+.vscode
+.idea
```

Comment:
We have existing templates for [`Global/VisualStudioCode.gitignore`](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore) and [`Global/JetBrains.gitignore`](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore). Please drop these rules.

```suggestion

```